apps/web/app/modules/Auth/Register.page.test.tsx

```diff
@@ -29,14 +29,16 @@ describe("Register page", () => {
     renderWith({ withQuery: true }).render(<RemixStub />);
 
     expect(
-      screen.getByRole("heading", { name: "Sign Up" })
+      screen.getByRole("heading", { name: "Sign Up" }),
     ).toBeInTheDocument();
   });
 
   it("submits the form with valid data", async () => {
     renderWith({ withQuery: true }).render(<RemixStub />);
 
     const user = userEvent.setup();
+    await user.type(screen.getByLabelText("firstName"), "Name");
```

Comment:
```suggestion
    await user.type(screen.getByLabelText("First name"), "Name");
```

Reply:
Oops, I noticed the mistake, but for some reason, I didn't push the changes.

Reply:
Already fixed ✅